cpp/open3d/ml/PyTorch/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ set(TORCH_OPS_CUDA_SOURCES
     "../impl/continuous_conv/ContinuousConvCUDAKernels.cu"
 )
 
-if(CUDA_ENABLED)
+if(BUILD_CUDA_MODULE)
```

Comment:
can we do a search of `CUDA_ENABLED` in the rest of the repo and clean them up as well?

Reply:
Done.

Reply:
Done.